### PR TITLE
Converted <ElementInfo /> to functional component OSRG Issue #147

### LINKED ISSUE #147: Convert <ElementInfo /> to functional component

### PR DESCRIPTION
**Pull request by amit-agrawals**
Fixes #147

### CODE REVIEW

File: src/components/periodic-table-page/element-info/ElementInfo.tsx [hunk 1]
```diff
@@ -11,11 +11,11 @@ interface IElementInfoDataEntryProps {
   unit?: string;
 }
 
-const ElementInfoDataEntry: React.StatelessComponent<IElementInfoDataEntryProps> = ({
+const ElementInfoDataEntry = ({
   name,
   value,
   unit,
-}) => {
+}: IElementInfoDataEntryProps) => {
```
Comment (mpeyper) [inline src/components/periodic-table-page/element-info/ElementInfo.tsx]: It's not required if you want to move on, but we've also been removing the `I` prefix on types on files we're touching.

**Review by mpeyper — APPROVED** [review]

**Comment by mpeyper** [comment]
The coverage report seems weird on this one (changing more than I would have expected from the changes made). Are you far behind `next` or something like that?

File: src/components/periodic-table-page/element-info/ElementInfo.tsx (hunk 1 above)
Comment (amit-agrawals) [reply src/components/periodic-table-page/element-info/ElementInfo.tsx]: Done :)

Comment (mpeyper) [reply src/components/periodic-table-page/element-info/ElementInfo.tsx]: Have you pushed it? It still says `IElementInfoDataEntryProps` and not `ElementInfoDataEntryProps` for me in this review

**Review by mpeyper — APPROVED** [review]

**Pull request merged**